pkg/operator/cassandra/controller/controller.go

```diff
@@ -139,9 +134,12 @@ func New(
 			}
 			cc.enqueueCluster(newCluster)
 		},
-		DeleteFunc: func(obj interface{}) {
-			// TODO: handle deletion
```

Comment:
during deletion do we need to do a walk through of all racks and scale down 1 by 1? or is that not needed?

Reply:
No, this is definitely not needed!
Scaling down means that data gets moved around as the cluster rebalances. 
We just want to be done with it, so deletion should be quick.